bin/update-version.js

```diff
@@ -23,6 +23,21 @@ const simpleGit = require("simple-git")(process.cwd());
 const time = new Date();
 const minuteOfDay = time.getUTCHours() * 60 + time.getUTCMinutes();
 
+/**
+ * Return the given string padded with 0s out to the given width.
```

Comment:
Maybe note that this can be replaced with `.padStart`when we upgrade to node 8?

Reply:
Sure